### DISCUSSION TITLE
Error while using interact server for SSRF in nuclei

### DISCUSSION THREAD

**Discussion by thecyberneh** [projectdiscovery/nuclei#2957] [Q&A]
While using nuclei for testing SSRF, it is giving error like ```[ERR] Could not decode interactions: readObjectStart: expect { or n, but found <, error found in #1 byte of ...|<!doctype h|..., bigger context ...|<!doctype html><html lang="en" data-adblockkey="MFw|...
```

Nuclei Version:- 2.7.1
Nuclei Template version:- 9.0.2

Check screenshot for more
Screenshot:- https://drive.google.com/file/d/1AFm7dfMklogV9KaiXr2EpvjKq---c2WQ/view?usp=sharing

**Comment by Mzack9999** [comment]
@thecyberneh Thanks for opening this issue. Unfortunately, I'm having some problems reproducing the described behavior. Could you provide a few more pieces of info to help us reproduce the issue:
- Full command line used
- Execution environment
- Example template, if any

Thanks!

**Comment by MetzinAround** [comment]
Marking as closed. Please reopen if you want more help.
